CMakeLists.txt

```diff
@@ -22,35 +22,58 @@ option(tests          "Enable testing."                         on)
 option(pythonBindings "Enable python bindings."                 on)
 option(compileDocs    "Compile c++11 documentation examples."   on)
 
+# Provides backwards compatibility and prevents MACOS_RPATH warning
+cmake_policy(VERSION 3.0)
+
```

Comment:
I think a better solution is to only touch this policy and set that to the old behaviour 

cmake_policy(SET CMP0042 OLD)